backend/infrahub/task_manager/task.py

```diff
@@ -83,9 +86,42 @@ async def _get_related_nodes(cls, db: InfrahubDatabase, flows: list[FlowRun]) ->
         return related_nodes
 
     @classmethod
-    async def _get_logs(cls, client: PrefectClient, flow_ids: list[UUID]) -> FlowLogs:
+    async def _get_logs(
+        cls, client: PrefectClient, flow_ids: list[UUID], log_limit: int | None, log_offset: int | None
```

Comment:
Since this funciton is only about the logs I think we cna remove the prefix `log_` from the parameters
- log_limit > limit
- log_offset > offset